package.json

```diff
@@ -64,8 +64,8 @@
     "loglevel": "^1.4.0",
     "moment": "^2.16.0",
     "node-fetch": "^1.6.3",
-    "node-pre-gyp": "^0.6.30",
-    "node-sass": "^4.13.1",
+    "node-pre-gyp": "^0.17.0",
+    "node-sass": "npm:sass@^1.58.0",
```

Comment:
`node-sass` was the cause of the problem. But that package was deprecated and replaced by `sass`, which was supposed to be a drop in replacement. However, `node-pre-gyp` was still referring to `node-sass`. So creating an alias turned out to be possible and a good solution imo.